### PR TITLE
Fixed link to the datasets

### PR DESCRIPTION
**Pull request by apawlik**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -54,8 +54,10 @@ for each level and a very brief introduction to plotting.
 
 ## Data
 
-Data files for the lesson are available here: ({{page.dataurl %}})[{{page.dataurl %}}]
+Data files for the lesson are available here: 
+[{{page.dataurl %}}]({{page.dataurl %}})
 
+Please download the **.csv* (Comma Separated Value) files.
```
Comment (fmichonneau) [inline index.md]: I think there is a missing `*` after csv.

Comment (fmichonneau) [inline index.md]: Maybe add that 4 csv files need to be downloaded (species, plots, surveys, and combined).

Comment (apawlik) [inline index.md]: I thought about listing them explicitly and then got lazy :-) I'll do that.
About the missing * it's actually not it's supposed to be '*.csv' in italics in MD (so the forst star is formatting, second is wildcard and third is closing the formatting.

Comment (fmichonneau) [inline index.md]: Ha! I see about the stars :star2: But I think the wildcard needs to be escaped then... :grimacing: Otherwise use `*.csv` instead?

**Comment by apawlik** [comment]
How about now?

**Comment by fmichonneau** [comment]
Thanks, it looks good! :+1:

**Pull request merged**